cfme/tests/configure/test_paginator.py

```diff
@@ -0,0 +1,138 @@
+    ('servers', None, 'ServerDiagnosticsCollectLogs', False),
+    ('servers', None, 'DatabaseTables', True),
+    ('servers', None, 'DatabaseIndexes', True),
+    ('servers', None, 'DatabaseSettings', True),
+    ('servers', None, 'DatabaseClientConnections', True),
+    ('servers', None, 'DatabaseUtilization', False),
+
+    ('regions', None, 'Details', False),
+    ('regions', None, 'ImportTags', False),
+    ('regions', None, 'Import', False),
+    ('regions', None, 'HelpMenu', False),
+    ('regions', None, 'Diagnostics', False),
+    ('regions', None, 'DiagnosticsZones', False),
+    ('regions', None, 'OrphanedData', False),
+    ('regions', None, 'Servers', True),
+    ('regions', None, 'ServersByRoles', False),
+    ('regions', None, 'RolesByServers', False),
+    ('zones', None, 'Details', False),
+    ('zones', None, 'SmartProxyAffinity', False),
+    ('zones', None, 'Diagnostics', False),
+    ('zones', None, 'ServersByRoles', False),
+    ('zones', None, 'Servers', False),
+    ('zones', None, 'CANDUGapCollection', False),
+    ('zones', None, 'RolesByServers', False),
+    ('zones', None, 'ZoneCollectLogs', False),
+
+
+    ('candus', None, 'Details', False),
+    ('category', Category, 'All', False),
+    ('map_tags', MapTags, 'All', False),
+    ('red_hat_updates', RedHatUpdates, 'Details', False),
+    ('analysis_profile', AnalysisProfile, 'All', True),
+    ('system_schedules', None, 'Add', False),
+    ('users', None, 'All', True),
+    ('groups', None, 'All', True),
+    ('roles', None, 'All', True),
+    ('tenants', None, 'All', True),
+]
+
+details_pages = [
+    ('users', None, 'Details', False),
+    ('groups', None, 'Details', False),
+    ('roles', None, 'Details', False),
+    ('tenants', None, 'Details', False),
+    ('analysis_profile', AnalysisProfile, 'Details', False),
+    ('system_schedules', None, 'All', True),
+    ('system_schedules', None, 'Details', False),
+    ('tag', Tag, 'All', False),
+]
+
+
+def check_paginator_for_page(view):
+    try:
+        panel = view.browser.element('//ul[@class="pagination"]')
+        return panel.is_displayed()
+    except NoSuchElementException:
+        return False
+
+
+@pytest.yield_fixture(scope='module')
+def schedule(appliance):
+    schedule = appliance.collections.system_schedules.create(
+        name=fauxfactory.gen_alphanumeric(),
+        description=fauxfactory.gen_alphanumeric()
+    )
+    yield schedule
+    schedule.delete()
+
+
+@pytest.mark.parametrize('place_info', general_list_pages,
+                         ids=['{}_{}'.format(set_type[0], set_type[2].lower())
+                              for set_type in general_list_pages])
+def test_paginator(appliance, place_info):
+    """
+        Check paginator is visible for config pages
+    """
+    place_name, place_class, place_navigation, paginator_expected_result = place_info
+    if appliance.version < '5.9' and place_navigation == 'HelpMenu':
+        pytest.skip('No Help Menu tab for 5.8 version')
```

Comment:
**OPTIONAL**
It can be uncollected.

Reply:
**REQUIRED:** I'd like an uncollect here, this test will never be expected to pass.